pkgs/top-level/all-packages.nix

```diff
@@ -9093,17 +9093,15 @@ let
   neard = callPackage ../servers/neard { };
 
   nginx = callPackage ../servers/http/nginx {
-    rtmp        = true;
-    fullWebDAV  = true;
-    syslog      = false; # the patch is not found
-    moreheaders = true;
+    modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders ];
```

Comment:
I can't find any reference on this, but if I'm right w.r.t. precedence of `rec` over `with` I think it's better to leave this as is to make it more future-proof, otherwise some package added later would have a chance to spontaneously break unrelated things.